lambdas/service/app.py

```diff
@@ -1273,21 +1273,19 @@ def get_summary():
                                              authentication=request.authentication)
 
 
-def manifest_route(*, fetch: bool, initiate: bool, put: bool = False):
+def manifest_route(*, fetch: bool, initiate: bool):
     return app.route(
         # The path parameter could be a token *or* an object key, but we don't
         # want to complicate the API with this detail
         ('/fetch' if fetch else '')
         + ('/manifest/files' if initiate else '/manifest/files/{token}'),
-        # FIXME: Remove `put` argument and derive method from `initiate`
-        #        https://github.com/DataBiosphere/azul/issues/5533
         # FIXME: Modify comment to reflect the actual idempotence of the request
         #        https://github.com/DataBiosphere/azul/issues/3271
         # The initial request is currently not idempotent, and POST would be a
         # more appropriate HTTP method to use. However, once a fix for #3271
         # lands, the initial request *will* be idempotent. We already use PUT,
         # in order to avoid having changing the REST API yet again.
```

Comment:
Looks like I overlooked the second FIXME in my PR for #3271.

```suggestion
        # The initial PUT request is idempotent.
```

Can you please make this change in a commit referencing that issue?